compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeBuilder.java

```diff
@@ -5478,10 +5479,16 @@ private BLangRawTemplateLiteral createRawTemplateLiteral(NodeList<Node> members,
         return literal;
     }
 
-    private BLangNode createRegExpTemplateLiteral(NodeList<Node> reSequences, Location location) {
+    private BLangNode createRegExpTemplateLiteral(NodeList<Node> reSequences, Location location, Token startBackTick,
+                                                  Token endBackTick) {
         BLangRegExpTemplateLiteral regExpTemplateLiteral =
                 (BLangRegExpTemplateLiteral) TreeBuilder.createRegExpTemplateLiteralNode();
-        regExpTemplateLiteral.reDisjunction = (BLangReDisjunction) createReDisjunctionNode(reSequences, location);
+        
+        Location reDisjunctionPos = reSequences.size() > 0 ? getPosition(reSequences.get(0),
+                reSequences.get(reSequences.size() - 1)) : getPosition(startBackTick, endBackTick);
```

Comment:
```suggestion
        Location reDisjunctionPos = getPosition(startBackTick, endBackTick);
```
What happen if we use this approach? We should get the same result right?

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/39466/commits/49b17111ddcdae21a63a3f284309f903bd11b8a8.